### DISCUSSION TITLE
can't install BeEF on arch

### DISCUSSION THREAD

**Discussion by dimsmaul** [ruby/rubygems#6830] [Q&A]
Unfortunately, an unexpected error occurred, and Bundler cannot continue.

First, try this link to see if there are any existing issue reports for this error:
https://github.com/rubygems/rubygems/search?q=You+don%27t+have+write+permissions+for+the+%2Fusr%2Flib%2Fruby%2Fgems%2F3.0.0+directory.&type=Issues

If there aren't any reports for this error yet, please fill in the new issue form located at https://github.com/rubygems/rubygems/issues/new?labels=Bundler&template=bundler-related-issue.md, and copy and paste the report template above in there.
==> ERROR: A failure occurred in build().
    Aborting...
 -> error making: beef-git-exit status 4
 -> Failed to install the following packages. Manual intervention is required:
beef-git - exit status 4

when i run 
yay -S beef-git

**Comment by duckinator** [comment]
This is a problem with either BeEF or the AUR package for it.

It looks like someone already reported the issue on the AUR: https://aur.archlinux.org/packages/beef-git

It also looks like that package hasn't been updated in 4 years, which is probably related to it not building anymore.
